### PR TITLE
Chore: add new GH action to codeowners file

### PR DESCRIPTION
**Pull request by zuchka**
adding a new action triggered the `codeowners` warning. This adds the two files to CODEOWNERS

### CODE REVIEW

**Review by tolzhabayev — APPROVED** [review]
should this be @zuchka or @grafana/grafana-community-support? 
I am fine with either. 

The codeowners check fails because the files do not exist, so ideally do not merge this one until the https://github.com/grafana/grafana/pull/63228 is merged, then rebase on the main and merge. 

Otherwise some people will have broken codeowners checks if they branch out after your merge this one.

**Review by natellium — APPROVED** [review]
👍

**Pull request merged**
